doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -608,6 +608,21 @@ Printing universes
    `:n:`@qualid` for named universes (e.g. `eq.u0`), and :n:`@string`
    for raw universe expressions (e.g. `"Stdlib.Init.Logic.1"`).
 
+   By default when printing a subgraph `Print Universes` attempts to
+   find and print the source of the constraints. This can be
+   controlled by providing `With Constraint Sources` or `Without
+   Constraint Sources`.
```

Comment:
```suggestion
   By default `Print Universes` tries to find and print
   the source of the constraints. If `Without Constraint Sources` is given,
   this information won't be shown.
```

"find and print" - if find always works, then just say "print" and drop "tries".  If it doesn't always work, it would be helpful to say a little about when it may fail to find the source.

You don't need to have both With and Without syntax.  Just the one that's not the default.

Reply:
"find" doesn't always work but I don't know what to say about the failures

The default depends on if we use subgraph, ie `Print Universes` -> default without, `Print Universes Subgraph (names)` -> default with

Reply:
It seems too clever to have both `Subgraph` and `With Constraint Source` trigger the printing the source info.  How about not having `Subgraph` trigger this?  While it changes the behavior, it seems like something users can easily adapt to without advance notice or a warning.  Do you have a feel for how often people want to see Subgraph with or without the sources?  Only if it's a 90+% to 10-% split would I consider keeping Subgraph as a trigger for the source info.

Reply:
I do think it would be pretty close to 100% of the time.